python/tests/test_account.py

```diff
@@ -564,6 +564,34 @@ def test_qr_join_chat(self, acfactory, lp):
         assert ch.id >= 10
         wait_securejoin_inviter_progress(ac1, 1000)
 
+    def test_qr_verified_group_and_chatting(self, acfactory, lp):
+        ac1, ac2 = acfactory.get_two_online_accounts()
+        lp.sec("ac1: create verified-group QR, ac2 scans and joins")
+        chat1 = ac1.create_group_chat("hello", verified=True)
+        assert chat1.is_verified()
+        qr = chat1.get_join_qr()
+        lp.sec("ac2: start QR-code based join-group protocol")
+        chat2 = ac2.qr_join_chat(qr)
+        assert chat2.id >= 10
+        wait_securejoin_inviter_progress(ac1, 1000)
+
+        lp.sec("ac2: read member added message")
+        msg = ac2.wait_next_incoming_message()
+        assert "added" in msg.text.lower()
```

Comment:
maybe we could also check if the message is actually encrypted. the ffi-api has no direct is-encrypted functions, but dc_msg_get_showpadlock() should return 1.

Reply:
done.